### ISSUE TITLE
Enhancement: Expand Playlist with Additional Songs

### ISSUE THREAD

**Issue by Siddharth-Dagar-25** [Siddharth-Dagar-25/Geet-Music-Player#2]
# Feature Request: Expand Playlist with Additional Songs

## Objective :
Our current playlist JSON object is an integral part of our project, but it's time to infuse it with fresh energy by incorporating new songs. Each addition should be meticulously crafted to include essential details such as the song name, artist, song length, and an associated cover image. This enhancement not only expands our playlist but also enhances user engagement and satisfaction.

### Enhancement:
The current playlist JSON object could benefit from an expansion with new songs. Each addition should include comprehensive details such as the ->
- Song Name
- Artist Name
- Genre
- Language (Hindi, English, Punjabi etc.)
- Song Length
- Google Drive URL of the song
- Associated cover image (Google Drive URL).

### Development and Contribution Process
- Fork the repository to begin working on this feature.
- Maintain the integrity of existing code:
 -- Do not alter existing variable names, indexes, or IDs.
 -- Utilize the existing data structures and variables wherever possible.
- Follow best practices for naming new variables and functions to ensure code clarity and maintainability.
- Thoroughly comment your code to explain the logic and functionality of the added features.
- Ensure that the new features do not negatively impact the existing functionalities of the music player.

### Final Steps
- Once development is complete, test the new features to verify they work as expected without introducing bugs or affecting the existing functionality.
- Submit your changes via a Pull Request (PR) to the original repository for review.

Your contribution will enhance the music listening experience by providing users with greater control over their playback options and ensuring uninterrupted enjoyment of their favorite songs.

**Comment by AnukratiMehta** [comment]
@Siddharth-Dagar-25 I'd like to work on this.

**Comment by Siddharth-Dagar-25** [comment]
> @Siddharth-Dagar-25 I'd like to work on this.

@AnukratiMehta 
Certainly! I'm glad to hear that you're interested in working on this project. Please feel free to send me an email at sid.dagar0101@gmail.com so we can continue our discussion and coordinate the next steps. Looking forward to hearing from you soon!
